Updated readme to reflect lessons learnt from Adpotion poc (#10)
govuk-one-login/observability-infrastructure: fargate/README.md

## Prerequisites

Ensure you run the latest VPC stack and SAM Deployment Pipeline stack versions. At time of writing, v2.0.7 and v2.36.0, respectively. You can check the version of stacks in the [CloudFormation Console](https://eu-west-2.console.aws.amazon.com/cloudformation/home?region=eu-west-2#/stacks) in the description column.

Your container needs to have access to the internet, so that it can send it's metrics to Dynatrace.

> In order to pull secrets from Secrets mamnager you VPC will need a vpc endpoint for secrets manager.

Reviewer: ```suggestion
To pull secrets from Secrets Manager, your VPC will need a VPC Endpoint for Secrets Manager.
```
